assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -120,7 +120,7 @@ const List = styled(Area).attrs({
   /*
    * These overrides are an exception - generally scrollbars should all
    * look the same. We do this only here because this scrollbar is always visible.
-   * /
+   */
```

Comment:
Wow, I wasn't able to open the editor anymore because of this, thanks for the fix!